styleguide/source/_patterns/00-atoms/00-intro/_layout-intro-styling.scss

```diff
@@ -20,5 +20,5 @@
 .layout-intro-styling blockquote code {
   display: block;
   border: 0;
-  padding: .5em;
-}
\ No newline at end of file
+  padding: 0.5em;
```

Comment:
I opened https://issues.ama-assn.org/browse/EWL-3909 as a follow up for the Scss linter.